src/io/sampling.jl

```diff
@@ -369,6 +373,19 @@ function _check_bounds(lower, upper)
     end
 end
 
+"""
+Check ptype for discrete variable types.
+
+Returns true if discrete, false otherwise.
+
+# Arguments
+- `ptype` : String representing variable type.
+"""
+function _check_discrete(p_type::String)
+    check = any(p_type .== DISCRETE_FACTOR_TYPES)
```

Comment:
I think the code is readable enough without using an intermediate variable.

```suggestion
    return any(p_type .== DISCRETE_FACTOR_TYPES)
```